pkg/util/span/frontier.go

```diff
@@ -428,6 +428,10 @@ func (f *Frontier) Entries(fn Operation) {
 //
 // Note: neither [a-b) nor [m, q) will be emitted since they fall outside the spans
 // tracked by this frontier.
+//
+// It's worth noting that only spans in the frontier that intersect with the passed in span will
+// get surfaced. For example: if frontier contains [a-b), [d-f), and the caller passes [b,e),
+// only [d,e), will surface.
```

Comment:
isn't this comment change the same as the note above?
Not that I mind it, but it seems to be saying the same thing.  Feel free to rephrase of course.

Reply:
ah, I added it because the example in the original comment contained a frontier with a single contiguous span, and I was initially unclear how a frontier behaved when it contained several disjoint spans. I will rephrase to make this clearer.